controller/machine/controller.go

```diff
@@ -199,6 +199,13 @@ func (m *Lifecycle) createOrUpdate(obj *v3.Machine) (*v3.Machine, error) {
 			sshUser = obj.Spec.AzureConfig.SSHUser
 		}
 		obj.Status.SSHUser = sshUser
+		nodeConfig := &v3.RKEConfigNode{
```

Comment:
@StrongMonkey we should set the role on the nodeConfig along with the rest of the params, but it is unclear where the role will be coming from. Most likely we would need to add a field to the machine spec